### ISSUE TITLE
setHTML without mixin

### ISSUE THREAD

**Issue by igolden** [zenoamaro/react-quill#112]
Hey there!

Thanks for bundling up Quill into React, we're working this into a production application right now!

I'm loading up the text editor with the contents of a file (this.props.body). Right now, it loads just fine when value={this.props.body}, but when I try and run setHTML or setEditorContents without mixins ([per this link](https://facebook.github.io/react/blog/2016/07/13/mixins-considered-harmful.html)), I can't find any functions.

So in short, this issue is about the following:

* is pure component support available for react-quill currently? Or are mixins my only options?

* what's the simplest way for me to set HTML during the lifecycle of my EditorComponent?

Thanks again for all your hard work! I'd be happy to help merge in 1.0 beta in the future, and/or squash some bugs.
